<ui version="4.0" stdsetdef="1" >
  <author>Mohammad Al-Turany/Sergey Linev</author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>TGo4CreateNewHistogram</class>
  <widget class="QDialog" name="TGo4CreateNewHistogram" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>369</width>
        <height>309</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Create New Histogram</string>
    </property>
    <property name="windowIcon" >
      <pixmap>:/icons/go4logo2_big.png</pixmap>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>3</number>
      </property>
      <property name="spacing" >
        <number>1</number>
      </property>
      <item rowspan="1" row="5" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
          <item>
            <widget class="QPushButton" name="CreateHistogram" >
              <property name="text" >
                <string>Create &amp;Local</string>
              </property>
              <property name="shortcut" >
                <string>Alt+L</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="CearteRemote" >
              <property name="text" >
                <string>Create &amp;Remote</string>
              </property>
              <property name="shortcut" >
                <string>Alt+R</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer8" >
              <property name="sizeHint" >
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="Close" >
              <property name="minimumSize" >
                <size>
                  <width>30</width>
                  <height>30</height>
                </size>
              </property>
              <property name="maximumSize" >
                <size>
                  <width>30</width>
                  <height>30</height>
                </size>
              </property>
              <property name="text" >
                <string/>
              </property>
              <property name="icon" >
                <pixmap>:/icons/delete.png</pixmap>
              </property>
              <property name="shortcut" >
                <string/>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item rowspan="1" row="3" column="0" colspan="2" >
        <widget class="QGroupBox" name="YAxisGroup" >
          <property name="title" >
            <string>Y-Axis</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>7</number>
            </property>
            <property name="spacing" >
              <number>3</number>
            </property>
            <item row="0" column="4" >
              <widget class="QLabel" name="TextLabel6_2" >
                <property name="text" >
                  <string>Max.</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="5" >
              <widget class="QLineEdit" name="Ymax" >
                <property name="text" >
                  <string>10</string>
                </property>
              </widget>
            </item>
            <item row="0" column="2" >
              <widget class="QLabel" name="TextLabel5_2" >
                <property name="text" >
                  <string>Min.</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="3" >
              <widget class="QLineEdit" name="Ymin" >
                <property name="text" >
                  <string>1</string>
                </property>
              </widget>
            </item>
            <item row="0" column="0" >
              <widget class="QLabel" name="TextLabel4_2" >
                <property name="text" >
                  <string>No. of Bins :</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="1" >
              <widget class="QLineEdit" name="YNoOfBins" >
                <property name="text" >
                  <string>10</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item rowspan="1" row="4" column="0" colspan="2" >
        <widget class="QGroupBox" name="ZAxisGroup" >
          <property name="title" >
            <string>Z-Axis</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>7</number>
            </property>
            <property name="spacing" >
              <number>3</number>
            </property>
            <item row="0" column="0" >
              <widget class="QLabel" name="TextLabel4_3" >
                <property name="text" >
                  <string>No. of Bins :</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="1" >
              <widget class="QLineEdit" name="ZNoOfBins" >
                <property name="text" >
                  <string>10</string>
                </property>
              </widget>
            </item>
            <item row="0" column="2" >
              <widget class="QLabel" name="TextLabel5_3" >
                <property name="text" >
                  <string>Min.</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="3" >
              <widget class="QLineEdit" name="Zmin" >
                <property name="text" >
                  <string>1</string>
                </property>
              </widget>
            </item>
            <item row="0" column="4" >
              <widget class="QLabel" name="TextLabel6_3" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>5</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>Max.</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="5" >
              <widget class="QLineEdit" name="Zmax" >
                <property name="text" >
                  <string>10</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item row="0" column="1" >
        <widget class="Q3ButtonGroup" name="HisClassType" >
          <property name="maximumSize" >
            <size>
              <width>32767</width>
              <height>40</height>
            </size>
          </property>
          <property name="title" >
            <string>Class</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>6</number>
            </property>
            <property name="spacing" >
              <number>1</number>
            </property>
            <item row="0" column="0" >
              <widget class="QRadioButton" name="TH1_b" >
                <property name="text" >
                  <string>TH1</string>
                </property>
                <property name="buttonGroupId" >
                  <number>1</number>
                </property>
              </widget>
            </item>
            <item row="0" column="1" >
              <widget class="QRadioButton" name="TH2_b" >
                <property name="text" >
                  <string>TH2</string>
                </property>
                <property name="buttonGroupId" >
                  <number>2</number>
                </property>
              </widget>
            </item>
            <item row="0" column="2" >
              <widget class="QRadioButton" name="TH3_b" >
                <property name="text" >
                  <string>TH3</string>
                </property>
                <property name="buttonGroupId" >
                  <number>3</number>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item rowspan="1" row="2" column="0" colspan="2" >
        <widget class="QGroupBox" name="GroupBox7" >
          <property name="title" >
            <string>X-Axis</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>7</number>
            </property>
            <property name="spacing" >
              <number>3</number>
            </property>
            <item row="0" column="0" >
              <widget class="QLabel" name="TextLabel4" >
                <property name="text" >
                  <string>No. of Bins :</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="1" >
              <widget class="QLineEdit" name="XNoOfBins" >
                <property name="text" >
                  <string>10</string>
                </property>
              </widget>
            </item>
            <item row="0" column="2" >
              <widget class="QLabel" name="TextLabel5" >
                <property name="text" >
                  <string>Min.</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="3" >
              <widget class="QLineEdit" name="Xmin" >
                <property name="text" >
                  <string>1</string>
                </property>
              </widget>
            </item>
            <item row="0" column="4" >
              <widget class="QLabel" name="TextLabel6" >
                <property name="text" >
                  <string>Max.</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="5" >
              <widget class="QLineEdit" name="Xmax" >
                <property name="text" >
                  <string>10</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item row="1" column="1" >
        <widget class="Q3ButtonGroup" name="HisType" >
          <property name="maximumSize" >
            <size>
              <width>32767</width>
              <height>40</height>
            </size>
          </property>
          <property name="title" >
            <string>Type</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>6</number>
            </property>
            <property name="spacing" >
              <number>1</number>
            </property>
            <item row="0" column="0" >
              <widget class="QRadioButton" name="S_b" >
                <property name="text" >
                  <string>S</string>
                </property>
                <property name="buttonGroupId" >
                  <number>1</number>
                </property>
              </widget>
            </item>
            <item row="0" column="1" >
              <widget class="QRadioButton" name="D_b" >
                <property name="text" >
                  <string>D</string>
                </property>
                <property name="buttonGroupId" >
                  <number>2</number>
                </property>
              </widget>
            </item>
            <item row="0" column="2" >
              <widget class="QRadioButton" name="F_b" >
                <property name="text" >
                  <string>F</string>
                </property>
                <property name="buttonGroupId" >
                  <number>3</number>
                </property>
              </widget>
            </item>
            <item row="0" column="3" >
              <widget class="QRadioButton" name="I_b" >
                <property name="text" >
                  <string>I</string>
                </property>
                <property name="buttonGroupId" >
                  <number>4</number>
                </property>
              </widget>
            </item>
            <item row="0" column="4" >
              <widget class="QRadioButton" name="C_b" >
                <property name="text" >
                  <string>C</string>
                </property>
                <property name="buttonGroupId" >
                  <number>5</number>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item rowspan="2" row="0" column="0" colspan="1" >
        <layout class="QVBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <layout class="QHBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <property name="spacing" >
                <number>6</number>
              </property>
              <item>
                <widget class="QLabel" name="TextLabel1" >
                  <property name="text" >
                    <string>Name</string>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="Spacer7" >
                  <property name="sizeHint" >
                    <size>
                      <width>20</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                  </property>
                </spacer>
              </item>
              <item>
                <widget class="QLineEdit" name="HisName" >
                  <property name="enabled" >
                    <bool>true</bool>
                  </property>
                  <property name="text" >
                    <string>c1</string>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QHBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <property name="spacing" >
                <number>6</number>
              </property>
              <item>
                <widget class="QLabel" name="TextLabel2" >
                  <property name="text" >
                    <string>Title</string>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="Spacer6" >
                  <property name="sizeHint" >
                    <size>
                      <width>20</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                  </property>
                </spacer>
              </item>
              <item>
                <widget class="QLineEdit" name="HisTitle" >
                  <property name="text" >
                    <string>title</string>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <connections>
    <connection>
      <sender>Close</sender>
      <signal>clicked()</signal>
      <receiver>TGo4CreateNewHistogram</receiver>
      <slot>reject()</slot>
    </connection>
    <connection>
      <sender>CreateHistogram</sender>
      <signal>clicked()</signal>
      <receiver>TGo4CreateNewHistogram</receiver>
      <slot>CreateLocalHist()</slot>
    </connection>
    <connection>
      <sender>CearteRemote</sender>
      <signal>clicked()</signal>
      <receiver>TGo4CreateNewHistogram</receiver>
      <slot>CreateRemoteHis()</slot>
    </connection>
    <connection>
      <sender>TH1_b</sender>
      <signal>toggled(bool)</signal>
      <receiver>YAxisGroup</receiver>
      <slot>setDisabled(bool)</slot>
    </connection>
    <connection>
      <sender>TH1_b</sender>
      <signal>toggled(bool)</signal>
      <receiver>ZAxisGroup</receiver>
      <slot>setDisabled(bool)</slot>
    </connection>
    <connection>
      <sender>TH2_b</sender>
      <signal>toggled(bool)</signal>
      <receiver>ZAxisGroup</receiver>
      <slot>setDisabled(bool)</slot>
    </connection>
  </connections>
</ui>
